wasp/src/main/java/com/orhanobut/wasp/NetworkStack.java

```diff
@@ -7,5 +7,5 @@
 
   <T> void invokeRequest(RequestCreator requestCreator, InternalCallback<T> waspCallback);
 
-  <T> T invokeRequest(RequestCreator requestCreator) throws Exception;
+  Object invokeRequest(RequestCreator requestCreator) throws Exception;
```

Comment:
T is always Response. Nothing else. 
I've updated the PR and replaced T with Response. Can you review it?